staging/src/k8s.io/client-go/Godeps/Godeps.json

```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "k8s.io/client-go",
-	"GoVersion": "go1.7",
-	"GodepVersion": "v74",
+	"GoVersion": "go1.8",
+	"GodepVersion": "v79",
```

Comment:
update-staging-client-go.sh forces godep v74, see this [line](https://github.com/kubernetes/kubernetes/blob/master/hack/update-staging-client-go.sh#L25). I suspect staging/copy.sh was run directly, however, that should give you a [warning](https://github.com/kubernetes/kubernetes/blob/master/staging/copy.sh#L49).